__tests__/app.test.js

```diff
@@ -137,4 +137,78 @@ describe('app.js', () => {
             })
         })
     })
+
+    describe('GET /api/articles/:article_id/comments', () => {
+        test('200: responds with an array of comments with the right properties', () => {
+            return request(app).get('/api/articles/1/comments')
+            .expect(200)
+            .then(({body}) => {
+                const {comments} = body
+                comments.forEach((comment) => {
+                    expect(comment.hasOwnProperty('comment_id')).toBe(true)
+                    expect(typeof comment.comment_id).toBe('number')
```

Comment:
In writing comment.comment_id .toBe('number') we are proving that the comment_id property exists. So we don't need to worry about the hasOwnProperty test as well :) same for all of the other properties.